matter/spec/cluster.go

```diff
@@ -90,6 +93,7 @@ func (s *Section) toClusters(d *Doc, entityMap map[asciidoc.Attributable][]types
 		c.AddBitmaps(bitmaps...)
 		c.AddEnums(enums...)
 		c.AddStructs(structs...)
+		c.TypeDefs = append(c.TypeDefs, typedefs...)
```

Comment:
I made changes to the hierarchy resolution code to better disambiguate types with the same name in different parts of the spec. I can update this after merge to do the same for typedefs.

Reply:
Sounds good, thank you!